src/ObjectUtils/CoffFile.cpp

```diff
@@ -662,9 +680,71 @@ CoffFileImpl::LoadExceptionTableEntriesAsSymbols() {
     symbol_info->set_address(unwind_range.start);
     symbol_info->set_size(unwind_range.end - unwind_range.start);
   }
+
+  if (module_symbols.symbol_infos().empty()) {
+    return ErrorMessage{
+        absl::StrCat(kErrorMessagePrefix, "not even a single address range found.")};
+  }
   return module_symbols;
 }
 
+ErrorMessageOr<ModuleSymbols> CoffFileImpl::LoadDynamicLinkingSymbolsAndUnwindRangesAsSymbols() {
+  static constexpr std::string_view kErrorMessagePrefix = "Unable to load fallback symbols: ";
+
+  ErrorMessageOr<std::vector<UnwindRange>> unwind_ranges_or_error = GetUnwindRanges();
+  if (!unwind_ranges_or_error.has_value()) {
+    return ErrorMessage{
+        absl::StrCat(kErrorMessagePrefix, unwind_ranges_or_error.error().message())};
+  }
+
+  ErrorMessageOr<ModuleSymbols> dynamic_linking_symbols =
+      LoadSymbolsFromExportTableInternal(unwind_ranges_or_error.value());
+  ErrorMessageOr<ModuleSymbols> unwind_ranges_as_symbols =
+      LoadExceptionTableEntriesAsSymbolsInternal(unwind_ranges_or_error.value());
+  if (!dynamic_linking_symbols.has_value() && !unwind_ranges_as_symbols.has_value()) {
+    return ErrorMessage{absl::StrFormat("%s1) %s 2) %s", kErrorMessagePrefix,
+                                        dynamic_linking_symbols.error().message(),
+                                        unwind_ranges_as_symbols.error().message())};
+  }
+  std::vector<SymbolInfo> fallback_symbols;
```

Comment:
From the name it is a bit confusing that those are essentially the symbols for the result.